tests/components/deconz/test_config_flow.py

```diff
@@ -372,84 +378,84 @@
     gateway = await setup_deconz_integration(hass)
 
     result = await hass.config_entries.flow.async_init(
-        config_flow.DOMAIN,
+        DOMAIN,
         data={
             ssdp.ATTR_SSDP_LOCATION: "http://2.3.4.5:80/",
-            ssdp.ATTR_UPNP_MANUFACTURER_URL: config_flow.DECONZ_MANUFACTURERURL,
+            ssdp.ATTR_UPNP_MANUFACTURER_URL: DECONZ_MANUFACTURERURL,
             ssdp.ATTR_UPNP_SERIAL: BRIDGEID,
         },
         context={"source": "ssdp"},
     )
 
-    assert result["type"] == "abort"
+    assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
     assert result["reason"] == "already_configured"
-    assert gateway.config_entry.data[config_flow.CONF_HOST] == "2.3.4.5"
+    assert gateway.config_entry.data[CONF_HOST] == "2.3.4.5"
 
 
 async def test_ssdp_discovery_dont_update_configuration(hass):
     """Test if a discovered bridge has already been configured."""
     gateway = await setup_deconz_integration(hass)
 
     result = await hass.config_entries.flow.async_init(
-        config_flow.DOMAIN,
+        DOMAIN,
         data={
             ssdp.ATTR_SSDP_LOCATION: "http://1.2.3.4:80/",
-            ssdp.ATTR_UPNP_MANUFACTURER_URL: config_flow.DECONZ_MANUFACTURERURL,
+            ssdp.ATTR_UPNP_MANUFACTURER_URL: DECONZ_MANUFACTURERURL,
             ssdp.ATTR_UPNP_SERIAL: BRIDGEID,
         },
         context={"source": "ssdp"},
     )
 
-    assert result["type"] == "abort"
+    assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
     assert result["reason"] == "already_configured"
-    assert gateway.config_entry.data[config_flow.CONF_HOST] == "1.2.3.4"
+    assert gateway.config_entry.data[CONF_HOST] == "1.2.3.4"
 
 
 async def test_ssdp_discovery_dont_update_existing_hassio_configuration(hass):
     """Test to ensure the SSDP discovery does not update an Hass.io entry."""
     gateway = await setup_deconz_integration(hass, source="hassio")
 
     result = await hass.config_entries.flow.async_init(
-        config_flow.DOMAIN,
+        DOMAIN,
         data={
             ssdp.ATTR_SSDP_LOCATION: "http://1.2.3.4:80/",
-            ssdp.ATTR_UPNP_MANUFACTURER_URL: config_flow.DECONZ_MANUFACTURERURL,
+            ssdp.ATTR_UPNP_MANUFACTURER_URL: DECONZ_MANUFACTURERURL,
             ssdp.ATTR_UPNP_SERIAL: BRIDGEID,
         },
         context={"source": "ssdp"},
     )
 
-    assert result["type"] == "abort"
+    assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
     assert result["reason"] == "already_configured"
-    assert gateway.config_entry.data[config_flow.CONF_HOST] == "1.2.3.4"
+    assert gateway.config_entry.data[CONF_HOST] == "1.2.3.4"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5028567)